lib/chartmogul/plan_groups/plans.rb

```diff
@@ -0,0 +1,15 @@
+module ChartMogul
+  module PlanGroups
```

Comment:
I feel it's a bit confusing to have `PlanGroups` in plural since this endpoint purpose is Plans for a single PlanGroup

Reply:
I did this because of this behavior:

```
irb(main):001:0> module ChartMogul
irb(main):002:1> class PlanGroup
irb(main):003:2> end
irb(main):004:1> end
=> nil
irb(main):005:0> ChartMogul::PlanGroup
=> ChartMogul::PlanGroup
irb(main):006:0> module ChartMogul
irb(main):007:1> module PlanGroup
irb(main):008:2> class Plans
irb(main):009:3> end
irb(main):010:2> end
irb(main):011:1> end
TypeError: PlanGroup is not a module
	from (irb):7:in `<module:ChartMogul>'
	from (irb):6
	from /usr/bin/irb:11:in `<main>'
irb(main):012:0>
```

In this example, the author seems to go with pluralizing for modules: https://boykoc.github.io/ruby/rubyonrails/2016/12/16/namespaces-and-modules-in-rails.html#where-i-landed